all.js

```diff
@@ -0,0 +1,199 @@
+// Index
+import './style.css';
+import ToDoList from './toDoList.js';
+
+const ToDoListArray = new ToDoList([]);
+
+const init = () => {
+  const toDoList = document.getElementById('to-do-list');
+  const toDoHeader = document.createElement('div');
+  toDoHeader.className = 'to-do-header';
+  toDoHeader.innerHTML = `<h4>Today's To Do</h4>
+    <div class="icon-container">
+    <?xml version="1.0" encoding="utf-8"?>
+    <svg fill="#000000" width="14px" height="14px" viewBox="0 0 24 24" id="update-alt" data-name="Flat Line" xmlns="http://www.w3.org/2000/svg" class="icon flat-line">
+      <path id="primary" d="M5.07,8A8,8,0,0,1,20,12" style="fill: none; stroke: rgb(0, 0, 0); stroke-linecap: round; stroke-linejoin: round; stroke-width: 2;"></path>
+      <path id="primary-2" data-name="primary" d="M18.93,16A8,8,0,0,1,4,12" style="fill: none; stroke: rgb(0, 0, 0); stroke-linecap: round; stroke-linejoin: round; stroke-width: 2;"></path>
+      <polyline id="primary-3" data-name="primary" points="5 3 5 8 10 8" style="fill: none; stroke: rgb(0, 0, 0); stroke-linecap: round; stroke-linejoin: round; stroke-width: 2;"></polyline>
+      <polyline id="primary-4" data-name="primary" points="19 21 19 16 14 16" style="fill: none; stroke: rgb(0, 0, 0); stroke-linecap: round; stroke-linejoin: round; stroke-width: 2;"></polyline>
+    </svg>
```

Comment:
I think these lines of code are not needed or can be replaced by something less complicated. Because of the YANGI principle (you aren't gonna need them) and also the KISS (keep it simple). Could it be possible to eliminate them or replace them with something shorter.